src/Wyam.Modules.AmazonWebServices/GenerateCloudSearchData.cs

```diff
@@ -0,0 +1,111 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Wyam.Common.Documents;
+using Wyam.Common.Meta;
+using Wyam.Common.Modules;
+using Wyam.Common.Pipelines;
+
+namespace Wyam.Modules.AmazonWebServices
+{
+    /// <summary>
+    /// Generates bulk JSON upload data to get documents into Amazon CloudSearch.
+    /// </summary>
+    /// <remarks>
+    /// This module creates a single document from a pipeline with JSON data containing the correctly formatted commands for Amazon CloudSearch.
+    /// Note that this just creates the document. Once that document is written to the file system, you will still need to upload the document
+    /// to a correctly configured CloudSearch instance using the API or Amazon CLI.
+    /// </remarks>
+    /// <category>Amazon Web Services</category>
```

Comment:
Let's set this category to "Content" since we're creating documents and setting their content. I'd like to try and fit all new modules into the existing categories. Eventually may add sub-categories too at which point we'd put AWS there.